rules/private/phases/phase_coverage_jacoco.bzl

```diff
@@ -15,14 +15,9 @@ def phase_coverage_jacoco(ctx, g):
     if not ctx.configuration.coverage_enabled:
         return
 
-    if _CodeCoverageConfiguration not in ctx.attr.scala:
```

Comment:
Is this early exit not longer relevant because the toolchain will always have the code coverage configuration?

Reply:
Not quite. This change is actually unrelated to toolchains. Previously, we included this phase in both the bootstrap and Zinc rules, despite this early exit preventing it from doing anything in the bootstrap rules.

I thought that was dumb, so I enabled the phase only for the Zinc rules and removed this statement.